components/eam/docs/tech-guide/clubb.md

```diff
@@ -0,0 +1,23 @@
+# Cloud Layers Unified By Binormals
+
+## Overview
+
+The Cloud Layers Unified By Binormals (CLUBB) parameterization is a parameterization of subgrid-scale turbulence and clouds.  It prognoses turbulent fluxes of heat, moisture, and momentum, and it diagnoses the liquid cloud fraction and liquid water mixing ratio.  To do so, it prognoses higher-order turbulence moments and closes those prognostic equations by use of an assumed double-Gaussian shape of the subgrid probability density function.  CLUBB operates throughout the troposphere, but it contributes especially to the planetary boundary layer and low-cloud regimes, including stratocumulus and shallow cumulus regimes.  
+
+### References
+
+- Larson (2022) [@larson_clubb-silhs_2022]
```

Comment:
Don't break out the references separately. Keep them inline as done in other sections like P3 and MAM.  You can put all of these after the first sentence.